pkg/v7/chartconfig/desired.go

```diff
@@ -0,0 +1,118 @@
+package chartconfig
+
+import (
+	"context"
+
+	"github.com/giantswarm/apiextensions/pkg/apis/core/v1alpha1"
+	"github.com/giantswarm/errors/guest"
+	"github.com/giantswarm/microerror"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/cluster-operator/pkg/label"
+	"github.com/giantswarm/cluster-operator/pkg/v7/key"
+)
+
+const (
+	chartConfigAPIVersion           = "core.giantswarm.io"
+	chartConfigKind                 = "ChartConfig"
+	chartConfigVersionBundleVersion = "0.3.0"
```

Comment:
This is a pain and I'd like to align with the release index eventually. For now just getting the updated resource / service structure sorted.